addons/sale/controllers/portal.py

```diff
@@ -168,54 +169,64 @@ def portal_order_page(self, order_id, report_type=None, access_token=None, messa
 
         return request.render('sale.sale_order_portal_template', values)
 
-    def _get_payment_values(self, order_sudo):
+    def _get_payment_values(self, order_sudo, tokens_included='partner_only', **kwargs):
```

Comment:
Suggestion for clarity because `tokens_included` sounds "boolean". In particular when you check `if tokens_included` below.
```suggestion
    def _get_payment_values(self, order_sudo, include_tokens_of='partner_only', **kwargs):
```